recipe/meta.yaml

```
@@ -53,11 +52,12 @@ outputs:
         - libpng
         # ffmpeg
         - numpy
+        # exclude 8.3.0 specifically due to pytorch/vision#4146, python-pillow/Pillow#5571
         - pillow >=4.1.1,<8.3
-        - pytorch >=1.8.0={{ torch_proc_type }}*
+        - pytorch
       run:
         - python
-        - pytorch >=1.8.0={{ torch_proc_type }}*
+        - pytorch
```

```suggestion
        - pytorch
        - pytorch-gpu    # [cuda_compiler_version != "None"]
        - pytorch-cpu    # [cuda_compiler_version == "None"]
```

This should work while we figure out how we want to pin.